spec/postgres_admin_spec.rb

```diff
@@ -58,20 +60,26 @@
         expect(author_count).to eq(2)
         expect(book_count).to   eq(3)
       end
+
+      after do
+        if defined?(PostgresRunner)
+          PostgresRunner.hard_reset
+        else
+          CiPostgresRunner.stop
+          CiPostgresRunner.start
```

Comment:
Super-minor, but you could add hard_reset method to CiPostgresRunner for consistency with PostgresRunner.

Reply:
Sure, I can do that in a followup.